Whiskey/Tasks/New-WhiskeyProGetUniversalPackage.ps1

```diff
@@ -228,7 +238,21 @@ function New-WhiskeyProGetUniversalPackage
                     }
 
                     Write-WhiskeyInfo -Context $TaskContext -Message $operationDescription
-                    Invoke-WhiskeyRobocopy -Source $sourcePath.trim("\") -Destination $destination.trim("\") -WhiteList $whitelist -Exclude $robocopyExclude | Write-WhiskeyVerbose -Context $TaskContext
+
+                    $robocopyOutputPath = Join-Path -Path $TaskContext.Temp -ChildPath ('RobocopyOutput.{0}.txt' -f [IO.Path]::GetRandomFileName())
+                    Invoke-WhiskeyRobocopy -Source $sourcePath.trim("\") -Destination $destination.trim("\") -WhiteList $whitelist -Exclude $robocopyExclude -LogPath $robocopyOutputPath | Out-Null
+                    if( $LASTEXITCODE -ge 8 )
+                    {
+                        $robocopyOutput = Get-Content -Path $robocopyOutputPath -Raw
+                        if ($robocopyOutput)
+                        {
+                            $robocopyOutput = $robocopyOutput.Split([Environment]::NewLine) | Where-Object { $_ -ne '' }
+                            $robocopyOutput | Write-WhiskeyVerbose -Context $TaskContext

```

Comment:
Write this as information. Since something went wrong, we want to show the output by default.

Reply:
Good catch, fixed.